docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,107 @@
+// This is a parameterized module. Parameters used:
+//
+// parameter-version-runtime-plugin: runtime-specific name for the runtime version property variable in the POM file.
+// parameter-artifact-name-runtime-plugin: runtime plugin name in the POM file.
+// parameter-group-name-runtime-plugin: runtime plugin Maven artifact group name.
+// parameter-runtime-name: Runtime specific atribute for substituting the approrpiate expanded rumtime name.
+// parameter-maven-plugin-command-run: the runtime-specific Maven plugin command used to run the application.
+// parameter-link-appdev-guide: link to the procedure providing a complete example of the appropriate runtime-specific POM file configuration.
+//  context: used in anchor IDs to conflicts due to duplicate IDs.
+//
+// Rationale: This procedure is the same for 2 or more runtimes.
+[id='deploying-an-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+Deploy your {parameter-runtime-name} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A Maven-based {parameter-runtime-name} application
+* The Red Hat Maven repository link:{parameter-link-appdev-guide}[specified] in the `pom.xml` of your project.
+* RHEL 7 installed.
+* OpenJDK 8 installed.
+* Maven 3.5 or later installed.
+
+
+.Procedure
+
+. Navigate to the directory with your application project:
++
+[source,bash,subs="attributes+",options="nowrap"]
+----
+$ cd myApp
+----
+
+. Ensure that the {parameter-runtime-name} Maven plugin is configured correctly in the `pom.xml` of your project:
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<project ... >
+  ifdef::built-for-spring-boot,built-for-spring-boot-1-5[]
+  ...
+  <!-- Specify target artifact type for the repackage goal. -->
+  <packaging>jar</packaging>
+endif::[]
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>{parameter-group-name-runtime-plugin}</groupId>
+        <artifactId>{parameter-artifact-name-runtime-plugin}</artifactId>
+        <version>{parameter-version-runtime-plugin}</version>
+ifdef::built-for-vertx[]
+        <executions>
+          <execution>
+            <id>vmp</id>
+            <goals>
+              <goal>initialize</goal>
+              <goal>package</goal>
+            </goals>
+          </execution>
+        </executions>
+endif::[]
+ifdef::built-for-thorntail[]
+        <executions>
+          <execution>
+            <goals>
+              <goal>package</goal>
+            </goals>
+          </execution>
+        </executions>
+endif::[]
+ifdef::built-for-spring-boot,built-for-spring-boot-1-5[]
+        <executions>
+          <execution>
+              <goals>
+               <goal>repackage</goal>
+             </goals>
+          </execution>
+        </executions>
+endif::[]
+      </plugin>
+    </plugins>
+  </build>
+  ...
+</project>
+----
+
+. Package and run your application:
```

Comment:
I pretty sure than mvn ${runtime}:run don't create a package.  It's is used only for the start app. Please check links 
[vertx-maven-plugin-doc](https://reactiverse.io/vertx-maven-plugin/#in-foreground-using-strong-a-href-vertx-run-vertx-run-a-strong)
[thorntail-maven-plugin-doc](https://docs.thorntail.io/2.2.1.Final/#using-thorntail-maven-plugin_thorntail#thorntail-maven-plugin-goals_thorntail)
[spring-boot-maven-plugin-doc](https://docs.spring.io/spring-boot/docs/current/maven-plugin/usage.html)

Reply:
Good catch! I runs the application but I think calling it packacing would be a stretch :)